locales/ru_RU/LC_MESSAGES/duckduckgo.po

```diff
@@ -104,7 +104,7 @@ msgstr "%1$sУзнайте больше%2$s."
 
 msgctxt "precise_user_location"
 msgid "%sLearn how we keep your location private%s."
-msgstr "%1$sУзнайте, как мы сохраняем конфиденциальность вашего местоположения%2$s."
+msgstr "%1$sКак мы сохраняем конфиденциальность вашего местоположения%2$s."
```

Comment:
![image](https://user-images.githubusercontent.com/63919734/179726398-c05c6e34-b591-4adc-848f-74008b9f204b.png)

Reply:
Thanks for raising this. I will forward the issue to my colleagues.